files_daily.sh

```diff
@@ -17,7 +17,7 @@ if [[ ! -v PROJECT_NAME ]]; then
 fi
 
 # check how many sites (ignore [^ name created by using pattern in mounts)
-NUMBER_OF_SITES=$(find $SITES_DIR -maxdepth 0 -type d ! -name '[^' | wc -l)
+NUMBER_OF_SITES=$(find $SITES_DIR -maxdepth 0 -type d | wc -l)
```

Comment:
use double quote to prevent globbing (also on next few lines)

Reply:
doesn't work here. Tried it already. if I used "" it won't find the directory